6 icons: AVR-Remote, lichess, Print, Location Share, Notification notes, MJ PDF Reader

Aircan opened the pull request:
First contribution. Hope that fits. Otherwise please let me know.
.svg files in /others and appfilters.xml file appended accordingly.

Donnnno commented:
Hi, thanks for your contribution, the icons look good!
A couple of things:
- we already have one for lichess
- can you make them white? #fff
- make sure that every line-width is 1px
- remove clipping masks from your icons

Aircan commented:
- lichess: Not part of my (latest) version, hence created. Shall I remove it from my repo., or will you "ignore"?
- white: Will do so
- 1 px: Double checked this (I thought). Which icon is affected?
- clipping: Which one is affected? Hints on how to?

After rework I will open another PR.

Donnnno commented:
You can just update this one ;) No need to create a new PR for some quick fixes

Aircan commented:
Updated accordingly, please check if your 4 bullet points are resolved.
Kept lichess in, sure you will ignore.

Aircan commented:
@Donnnno I was almost creating an icon for LibreTube, since this is not available on my device, although having installed <latest>. But in appfilter.xml the actual app info is entered here. So, a bug? Or part of main branch, but not yet released/delivered?

Donnnno commented:
I'll take a look

Aircan commented:
Removed lichess icon and updated entry in appfilter.xml. Obviously there was a name change.
What is the conflict about, that need to be manually solved? Next time I can then consider that instantly? Is it about alphabetical order?

Aircan commented:
@Donnnno anything left open, that hinders the merge?
When do you plan the next release?

Donnnno commented on other/appfilter.xml:
```diff
@@ -7024,7 +7041,7 @@
     <!-- Life Fits Into Home -->
     <item component="ComponentInfo{tr.gov.saglik.hayatevesigar/com.turktelekom.guvenlekal.ui.activity.SplashActivity}" drawable="life_fits_into_home" />
 
-    <!-- Liftago -->
```
can you re-add the > here?

Aircan replied:
me? In my forked appfilter.xml, everything looks fine without missing >

Donnnno commented:
Hi, I hope to create a new release soon. I'm pretty busy with lots of different stuff, but after that, I have time to update it.

Donnnno commented:
Hi, I had to remake all the icons.

I noticed there were images embedded, there elements were erased or something? 

![image](https://user-images.githubusercontent.com/31142286/199676970-83c42438-63c1-48e5-8488-fb79551ccb3a.png)

And had to change the names because there are no capitals or spaces allowed
